packages/core/src/createGetComputedCss.js

```diff
@@ -90,11 +93,17 @@ const createGetComputedCss = (config) => {
 				/** Whether the current style is a condition (i.e. media or supports query). */
 				const isCondition = name.charCodeAt(0) == 64
 
+				/** Whether the current style is a CSS selector */
+				const isSelector = !selectors.length || captureSelector.test(name)
```

Comment:
the `!selectors.length` code is to detect `global` styles, where all top level keys are treated as literal CSS selectors